.idea/.name

```diff
@@ -0,0 +1 @@
+Fingerprint-Android
```

Comment:
Is there a reason to have `.idea` directory in the repository? Usually it's excluded from Git.

Reply:
At some projects it can be useful, it provides an ability to share code-style config etc. But in this particular case the code style is kotlin default, so you're right, the whole directory is useless, and can cause merge-conflicts. Removed, thanks.

Reply:
Is think `.idea` isn’t a good solution for code style management: it contains personal settings that don’t affect the code, therefore it changes too often, it’s not designed to be read by humans, and other developers may use other IDEs. Probably, some of the points can be solved by including into Git only the files related to the code style. In web development we use other tools to control code style, e.g. EditorConfig and ESLint. The tools are IDE-independent and can run in CI.

I’m not sure if it all is relevant for Android development. As far as I can see, you had added only few files from `.idea`. If this is a standard practice in Android development, then it’s ok for me. I just wanted to know why you had done it that way.